tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -774,3 +691,114 @@ IncludeByKey.SSP0_MIXER_2LEVEL {
 		}
 	"1"	"platform/intel/nocodec-ssp0-2level.conf"
 }
+
+
+# There is pinmux conflict between SSP1 and DMIC on MTL,
+# so include SSP1 pipelines conditionally.
+IncludeByKey.SSP1_ENABLED {
+	"true" {
+		Object.Dai.SSP [
+			{
+				id 		1
+				dai_index	1
```

Comment:
Can we not do this via a macro defining the SSP to use in the platform file i.e.
```
# mtl.conf only define the SSPs that are available
undefine NOCODEC_TEST_SSP1
define NOCODEC_TEST_SSP2
define NOCODEC_TEST_SSP3
define NOCODEC_TEST_SSP4
# and so on
```

Reply:
I don't think topology2 support 'undefine' keyword. The only thing we can do is to first set the default values for each DAI and override default values in PLATFORM.conf